packages/contracts/contracts/L1/SecurityCouncil.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { MultiSigWallet } from "../universal/MultiSigWallet.sol";
+import { Semver } from "../universal/Semver.sol";
+
+/**
+ * @custom:proxied
+ *
+ * @title SecurityCouncil
+ *
+ * @notice SecurityCouncil receives validation requests for specific output data,
+ *         and allows security council parties to validate & agree on transactions before execution.
+ */
+contract SecurityCouncil is MultiSigWallet, Semver {
+    /**
+     * @notice A mapping of output info array which has been submitted to validate.
+     */
+    mapping (uint256 => OutputInfo) public outputInfos;
+
+    /**
+     * @notice Emitted when a validation request is submitted.
+     *
+     * @param transactionId Index of the submitted transaction.
+     * @param outputRoot    L2 output root that was proven aginst.
+     * @param l2BlockNumber The L2 block number of the output root.
+     */
+    event Validation(
+        uint256 indexed transactionId,
+        bytes32 outputRoot,
+        uint128 l2BlockNumber
+    );
+
+    /**
+     * @notice The address of the colosseum contract. Can be updated via upgrade.
+     */
+    address public COLOSSEUM;
```

Comment:
Why did you change it into non `immutable`?

Reply:
The purpose is to make initialization more consistent while avoiding overriding the parent contract's initialize function.